api/api/admin/__init__.py

```diff
@@ -47,32 +57,214 @@ class MediaReportAdmin(admin.ModelAdmin):
         "reason",
     )
     list_display_links = ("id",)
+    list_select_related = ("media_obj",)
     search_fields = ("description", "media_obj__identifier")
     autocomplete_fields = ("media_obj",)
     actions = None
+    media_type = None
+
+    @admin.display(description="Has sensitive text")
+    def has_sensitive_text(self, obj):
+        """
+        Return `True` if the item cannot be found in the filtered index - which means the item
+        was filtered out due to text sensitivity.
+        """
+        if not self.media_type or not obj:
+            return None
+
+        filtered_index = f"{settings.MEDIA_INDEX_MAPPING[self.media_type]}-filtered"
+        try:
+            search = (
+                Search(index=filtered_index)
+                .query("term", identifier=obj.media_obj.identifier)
+                .execute()
+            )
+            if search.hits:
+                return False
+        except NotFoundError:
+            logging.error(f"Could not resolve index {filtered_index}")
+            return None
+        return True
+
+    @admin.display(description="Title and creator")
+    def title(self, obj):
+        title = obj.media_obj.title or "Unnamed media"
+
+        if not (obj.media_obj.creator or obj.media_obj.creator_url):
+            return title
+
+        if obj.media_obj.creator and obj.media_obj.creator_url:
+            return mark_safe(
+                f"{title} by {create_link(obj.media_obj.creator_url, obj.media_obj.creator)}"
+            )
+        else:
+            creator = obj.media_obj.creator or obj.media_obj.creator_url
+            creator_string = f" by {creator}" if creator else ""
+            return f"{title}{creator_string}"
+
+    def media_display(self, obj):
+        pass
```

Comment:
At the very least it should raise `NotImplementedError` (referring to @AetherUnbound's comment regarding this being abstract).

```suggestion
    @abc.abstractmethod
    def media_display(self, obj):
        raise NotImplementedError("Subclasses must implement `media_display`")
```